bodhi/__init__.py

```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
We've been using this on fmn where we also have split the main module into multiple projects.